lib/flatpak-installation.c

```diff
@@ -1976,6 +1977,41 @@ flatpak_installation_uninstall (FlatpakInstallation    *self,
                                 gpointer                progress_data,
                                 GCancellable           *cancellable,
                                 GError                **error)
+{
+  return flatpak_installation_uninstall_full (self, FLATPAK_UNINSTALL_FLAGS_NONE,
+                                              kind, name, arch, branch,
+                                              progress, progress_data,
+                                              cancellable, error);
+}
+
+/**
+ * flatpak_installation_uninstall_full:
+ * @self: a #FlatpakInstallation
+ * @flags: set of #FlatpakUninstallFlags flag
+ * @kind: what this ref contains (an #FlatpakRefKind)
```

Comment:
No need to mention this is a `FlatpakRefKind`; gtk-doc shows the type already, separately.

Reply:
This comes from the original `flatpak_installation_uninstall()` method, not really my wording.